diff --git a/dist/preview-api/index.js b/dist/preview-api/index.js
index d7788050671535b3a88d7a3879b70f7eb210988e..6f0ad2d1435dd92a70262425d75ec4a589b70c7c 100644
--- a/dist/preview-api/index.js
+++ b/dist/preview-api/index.js
@@ -5812,7 +5812,7 @@ var Kr = class Kr {
         F.reason && m.add(F.reason);
       }, "onUnhandledRejection");
       if (this.renderOptions.autoplay && r && g && this.phase !== "errored") {
-        window.addEventListener("error", b), window.addEventListener("unhandledrejection", S), this.disableKeyListeners = !0;
+        window?.addEventListener?.("error", b), window?.addEventListener?.("unhandledrejection", S), this.disableKeyListeners = !0;
         try {
           if (E ? await g(y) : (y.mount = async () => {
             throw new bc({ playFunction: g.toString() });
@@ -5829,7 +5829,7 @@ var Kr = class Kr {
         if (!P && m.size > 0 && this.channel.emit(
           gc,
           Array.from(m).map(Ci)
-        ), this.disableKeyListeners = !1, window.removeEventListener("unhandledrejection", S), window.removeEventListener("error", b), x.aborted)
+        ), this.disableKeyListeners = !1, window?.removeEventListener?.("unhandledrejection", S), window?.removeEventListener?.("error", b), x.aborted)
           return;
       }
       await this.runPhase(x, "completing", async () => {
@@ -5896,7 +5896,7 @@ var Kr = class Kr {
       }
       await new Promise((r) => setTimeout(r, 0));
     }
-    window.location.reload(), await new Promise(() => {
+    window?.location?.reload?.(), await new Promise(() => {
     });
   }
 };
