diff --git a/node_modules/@storybook/api/dist/esm/modules/refs.js b/node_modules/@storybook/api/dist/esm/modules/refs.js
index 49f2aba..b441b81 100644
--- a/node_modules/@storybook/api/dist/esm/modules/refs.js
+++ b/node_modules/@storybook/api/dist/esm/modules/refs.js
@@ -73,7 +73,7 @@ export var getSourceType = function getSourceType(source, refId) {
   var localOrigin = location.origin,
       localPathname = location.pathname;
 
-  var _URL = new URL(source),
+  var _URL = new URL(source || location.origin),
       sourceOrigin = _URL.origin,
       sourcePathname = _URL.pathname;
 
