diff --git a/node_modules/telejson/dist/cjs/index.js b/node_modules/telejson/dist/cjs/index.js
index d32dbcf..d2f1fd9 100644
--- a/node_modules/telejson/dist/cjs/index.js
+++ b/node_modules/telejson/dist/cjs/index.js
@@ -39,8 +39,6 @@ function _iterableToArrayLimit(arr, i) { if (typeof Symbol === "undefined" || !(
 
 function _arrayWithHoles(arr) { if (Array.isArray(arr)) return arr; }
 
-var isRunningInBrowser = typeof window !== 'undefined' && typeof window.document !== 'undefined'; // eslint-disable-next-line @typescript-eslint/ban-types, no-use-before-define
-
 var isObject = _isobject["default"];
 
 var removeCodeComments = function removeCodeComments(code) {
@@ -134,7 +132,7 @@ function convertUnconventionalData(data) {
   // Plus we need to check if running in a browser to ensure `Event` exist and
   // is really the dom Event class.
 
-  if (isRunningInBrowser && data instanceof Event) {
+  if (typeof Event !== "undefined" && data instanceof Event) {
     result = (0, _domEvent.extractEventHiddenProperties)(result);
     wasMutated = true;
   }
